### PR TITLE
fix #6 Removed Impure Helper and replaced it with a proper Method

### LINKED ISSUE #6: Uglify cannot handle the "impure helper"
Hello, 

Me again.

I'm using Laravel-Elixir to transpile and compress/minify my app. Looks like Uglify is not able to parse your code. I could successfully replace the helper function by a method, by copy/pasting the helper into a `methods: {}` section (same level as `mounted() {}`) and replacing the call to `replaceElementWith` (line 20) by `this.replaceElementWith`, and it did the trick.

Uglify still can't parse the code, but it seems to be related to medium-editor and not to your package. 

I'll be glad to submit a PR if you wish :)

### PR DESCRIPTION
**Pull request by dsampaolo**
Uglify is not able to minify the JS due to the "impure helper" used. I replaced it with a method inside the component.

### CODE REVIEW

File: medium-editor.js [hunk 1]
```diff
@@ -44,4 +37,13 @@ export default {
     this.$root.mediumEditor.removeElements(this.$refs.element)
   }
 
+  methods: {
+
+	// impure helper function, replaces some element from the dom with a new one of the given tag name
+	// returns the new one.
+	function replaceElementWith (element, tagName) {
```
Comment (janwirth) [inline medium-editor.js:44]: First off, thank you very much for your help :)

Secondly: is the function keyword really required? As you can see, the other member function keywords are not written out. Let's keep that consistent :)

If you want, you may add yourself to some contributor list in the readme.

**Comment by janwirth** [comment]
I can not reproduce this. An uglified version of the module is used on https://franzskuffka.github.io/vue-medium-editor/.

**Comment by javisperez** [comment]
Just for the record, I'm having this same this exact same error with Laravel-mix when it tries to uglify it. I hope @dsampaolo applies your comments and fix this PR, I'm needing this too :-\

**Comment by janwirth** [comment]
@javisperez @dsampaolo I think your problem is not actually this plugin. I do not think it makes sense to structure this plugin for the sake of making uglify happy.

There is a simple solution, called babili. It will make sure your ES6 does not get maimed during minification.
https://github.com/babel/babili

**Pull request closed without merging**